scss/6-components/_on-this-page.scss

```diff
@@ -63,14 +55,17 @@
 ///
 .cads-on-this-page--expandable {
   .cads-on-this-page__list {
-    columns: 1;
```

Comment:
Removing this avoids any column bugs for this component variant. We don't need to use column styles at all.